CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.11.17
+
+<!-- generated comparing v5.11.16..master -->
+
+_Apr 11, 2023_
+
+A big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Added theme scope for using multiple design systems (#36664) @siriwatknp
```

Comment:
Would be great if we can link the guide to this.

Reply:
Also, this feature in my opinion deserves a bump in the major version, so ideally this new version would be 5.12.0